driver/main.c

```diff
@@ -1882,37 +1954,6 @@ static int record_event_consumer(struct ppm_consumer_t *consumer,
 	ASSERT(head <= consumer->buffer_bytes_dim);
 	ASSERT(delta_from_end < consumer->buffer_bytes_dim + (2 * PAGE_SIZE));
 	ASSERT(delta_from_end > (2 * PAGE_SIZE) - 1);
-#ifdef _HAS_SOCKETCALL
```

Comment:
No more specific management for socketcall, since it is now immediately resolved to its internal syscall.